docs/curation-guidelines.md

```diff
@@ -63,23 +63,41 @@ To evaluate the discovered licenses for a package component, use the following s
     1. If there is/are SPDX-identified license(s) in the file, use the SPDX license expression to capture those licenses.
     2. If there is no license in the file, code `NONE`.
     3. If there is a license, but that license does not have an SPDX identifier, code `OTHER`.
-3. If the definition indicates a source location, curate the license information found on source component’s definition. Because of the flow through of license obligations from source to a binary package, the information from the source location’s core facet should be encoded as a “discovered license” in the package.
+3. Follow the above steps for the source component definition that corresponds to the package component definition.  If you see `</>` next to the magnifying glass symbol, you can pull it up automatically. 
 
-## Troubleshooting: sources of truth for the declared license
+### Canonical locations for the declared license for source definitions 
+When curating the declared license for a source code repository, look in the canonical locations listed below.  
+1.	**License file.**  Look to see if a license file (e.g. LICENSE, COPYING) is included in the specific commit or version you are curating.  Sometimes the license is found on the README.  There may be more than one declared license.  If there is not any license information at the specific commit or version level, check the master branch.
+2.	**File Headers.**  If there is not a declared license anywhere in the repository, look to see if there is any license information in the file headers.  Look at multiple files to determine if they are under different licenses.  You can also try using a search function to find any mention of ‘license,’ for example.
+
+### Canonical locations for the declared license for package definitions
+When curating the declared license for a package, look in all the canonical locations listed below (1-3).  **Appendix A** has canonical locations for common package ecosystems.  
+
+If the canonical locations do not match (e.g., the license file states Apache-2.0 and the metadata states MIT), contact the author/publisher.  If you do not receive a response after 7 days, prioritize curating the declared license in the order listed below.
+1.	**License file.**  Look to see if a license file (e.g. LICENSE, COPYING) is included in the package file contents.  Sometimes the license is found on the README.
+2.	**Package metadata.**  Look to see if the target component metadata includes any license info, such as a license identifier or license file URL. 
+3.	**Source repository.**  Look in the associated source repository.
+    * Look for version in source repository corresponding to target component version and use license information available there in any source canonical locations [LINK].
```

Comment:
Suggest removing "there" in "license information available there in"

Reply:
@royaljust - I simplified row 79 - please take a look and let me know what you think.  Thanks